run.sh

```diff
@@ -2,6 +2,47 @@
 
 set -o errexit
 
+usage() {
+    echo "deploy               -- Deploy airship"
+    echo "update_openstack     -- Update openstack deployment"
+    echo "add_compute          -- Add compute node"
```

Comment:
Add OpenStack compute node. I think more specific is better since the description is pretty short here.

Reply:
I'm wondering, `add_compute` does not need any argument? So any data for a new compute node need to be provided in some extra place? 
Maybe it could be mentioned...

Reply:
> I'm wondering, `add_compute` does not need any argument? So any data for a new compute node need to be provided in some extra place?
> Maybe it could be mentioned...

Yes will add more info.

Reply:
> Add OpenStack compute node. I think more specific is better since the description is pretty short here.

Yes will fix it.